citations/ITKBibliography.bib

```diff
@@ -0,0 +1,324 @@
+issn = {00942405},
+journal = {Medical Physics},
+month = {nov},
+number = {11},
+pages = {5197--5207},
+publisher = {John Wiley and Sons Ltd.},
+title = {{Introduction of a pseudo demons force to enhance deformation range for robust reconstruction of super-resolution time-resolved 4DMRI}},
+volume = {45},
+year = {2018}
+}
+@article{Malkasian2018a,
+author = {Malkasian, Shant and Hubbard, Logan and Dertli, Brian and Kwon, Jungnam and Molloi, Sabee},
+doi = {10.1016/j.jcct.2018.06.006},
+issn = {1876861X},
+journal = {Journal of Cardiovascular Computed Tomography},
+month = {sep},
+number = {5},
+pages = {425--435},
+publisher = {Elsevier Inc.},
+title = {{Quantification of vessel-specific coronary perfusion territories using minimum-cost path assignment and computed tomography angiography: Validation in a swine model}},
+volume = {12},
+year = {2018}
+}
+@article{Khanal2017a,
+author = {Khanal, Bishesh and Ayache, Nicholas and Pennec, Xavier},
+doi = {10.3389/fnins.2017.00132},
+issn = {1662453X},
+journal = {Frontiers in Neuroscience},
+month = {mar},
+number = {MAR},
+publisher = {Frontiers Research Foundation},
+title = {{Simulating longitudinal brain MRIs with known volume changes and realistic variations in image intensity}},
+volume = {11},
+year = {2017}
+}
+@article{Klemm2017,
+author = {Klemm, Martin and Kirchner, Thomas and Gr{\"{o}}hl, Janek and Cheray, Dominique and Nolden, Marco and Seitel, Alexander and Hoppe, Harald and Maier-Hein, Lena and Franz, Alfred M.},
+doi = {10.1007/s11548-016-1488-y},
+issn = {18616429},
+journal = {International Journal of Computer Assisted Radiology and Surgery},
+month = {mar},
+number = {3},
+pages = {351--361},
+publisher = {Springer Verlag},
+title = {{MITK-OpenIGTLink for combining open-source toolkits in real-time computer-assisted interventions}},
+volume = {12},
+year = {2017}
+}
+@article{Hadj-Hamou2016a,
+author = {Hadj-Hamou, Mehdi and Lorenzi, Marco and Ayache, Nicholas and Pennec, Xavier},
+doi = {10.3389/fnins.2016.00236},
+issn = {1662453X},
+journal = {Frontiers in Neuroscience},
+month = {jun},
+number = {JUN},
+publisher = {Frontiers Research Foundation},
+title = {{Longitudinal analysis of image time series with diffeomorphic deformations: A computational framework based on stationary velocity fields}},
+volume = {10},
+year = {2016}
+}
+@misc{Avants2015,
+author = {Avants, Brian B. and Johnson, Hans J. and Tustison, Nicholas J.},
+booktitle = {Frontiers in Neuroinformatics},
+doi = {10.3389/fninf.2015.00005},
+issn = {16625196},
+month = {mar},
+number = {MAR},
+publisher = {Frontiers Research Foundation},
+title = {{Neuroinformatics and the the insight toolkit}},
+volume = {9},
+year = {2015}
+}
+@inproceedings{Yatabe2017,
+author = {Yatabe, Marilia and Ruellas, Antonio and Gomes, Liliane and Macron, Lucie and Lopinto, Julia and Paniagua, Beatriz and Budin, Francois and Cevidanes, Lucia},
+booktitle = {2017 IADR/AADR/CADR General Session (San Francisco, California) },
```

Comment:
Unnecessary whitespace before right curly brace.

Reply:
What space? I cant see anything... :confused:

Reply:
`) }` space between closing parenthesis and brace, at the end of the line

Reply:
Exactly. Thanks @dzenanz . Sorry for not having been more explicit.

Reply:
wow, you guys have :eyes: 
Thanks for the clarification, fixed now (I think)

Reply:
:+1: Thanks for having taken care of this @bpaniagua . Addressed in 9cee7c1.